files/config.tsf

```diff
@@ -81,15 +81,6 @@ class SwaggerError extends Error {
   isApiException = true;
 }
 
-export interface SwaggerResponse<R> extends AxiosResponse<R> {}
```

Comment:
Do not remove SwaggerResponse. if response changed in interceptors we need to change the response type of API to this here we enable to change response type.

Reply:
Right, I guess I got a bit too eager with the deletes..

Reply:
SwaggerResponse is back, and I added a bit of example code to the response interceptor